lib/test.js

```diff
@@ -134,7 +138,11 @@ Test.prototype.run = function () {
 	}.bind(this));
 };
 
-Test.prototype.end = function () {
+Test.prototype.end = function (err) {
+	if (arguments.length >= 1 && err) {
```

Comment:
To avoid conflicts with `t.plan()`, `this.ifError(err)` could be replaced with:

```js
this.assertError = new assert.AssertionError({
  actual: err,
  message: 'Callback called with an error → ' + err,
  operator: 'callback'
});

this.exit();
```

This is the same stuff we do when promise rejects.